src/docs/05-go-client/02.5-starting-workflows.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Starting workflows
+permalink: /docs/go-client/start-workflows
+---
+
+# Starting workflows
+
+Starting workflows can be done from any service that can send requests to
```

Comment:
I might add a separate paragraph just for jitters. Explaining why batch workflows are a problem such as 
- Cadence not knowing about the intent, parallel progress vs queued jobs
- sudden load increase in DB, and issues deriving from it

You might want to add some examples to how to use jitter with options.

Reply:
I have added a section on JitterStart specifically, what do you think?